tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/TestApplication.java

```diff
@@ -51,7 +53,7 @@ public static WebArchive createDeployment() {
         return ShrinkWrap.create(WebArchive.class);
     }
 
-    @Test
+    @Test(groups = TEST_GROUP_REST)
```

Comment:
Forcing new tests to specify group is a big ask.

Reply:
I have changed this now so 'optional-tests' is the only group and the ReadMe is up to date